src/bessel.jl

```diff
@@ -590,6 +590,33 @@ function bessely(n::Integer, x::BigFloat)
     return z
 end
 
+"""
+    sphericalbesselj(nu, x)
+
+Spherical bessel function of the first kind at order `nu`, ``j_ν(x)``. This is the non-singular
+solution to the radial part of the Helmholz equation in spherical coordinates.
+"""
+function sphericalbesselj(nu, x::T) where {T}
+    if isapprox(x, 0, atol=1e-100)
```

Comment:
Better to just do `abs(x) ≤ 1e-100`

However, I'm suspicious of this criterion.  Where did you come up with the `1e-100` threshold?

Reply:
> However, I'm suspicious of this criterion. Where did you come up with the 1e-100 threshold?

I chose the cutoff at `1e-100` because Mathematica warned me about precision losses with starting around `1e-80` since Mathematica at least needed to square `x` during the calculation, so I figured that `1e-100` was a relatively safe cutoff. 

Perhaps a better way to do this would be `if abs(x) ≤ √eps(typeof(x)(0.0))`? That way, the cutoff scales with the precision used?